corehq/apps/app_manager/models.py

```diff
@@ -986,9 +986,71 @@ def validate(self, value, required=True):
 
 
 class NavMenuItemMediaMixin(DocumentSchema):
+    """
+        Language-specific icon and audio.
+        Properties are map of lang-code to filepath
+
+        Convention (for old docs): Unless a specific language key exists,
+        by default, media for that language is same as that for English
```

Comment:
I planned to deal with this almost at the end after getting everything in place. This is what I am going to do.
Since this class doesn't have access to app's default-language, I will add a special key called 'default' instead of 'en'. The behaviour will be as following
- All migrated old-docs will have this `default` key pointing to the media that was already set
- If a particular lang key doesn't exist the `default` media is returned. 
- One can set a lang-specific media, by editing icon/image while viewing app in that language
- If app's default language changes, the media pointed by `default` will be changed to that.